lib/src/controls/form/combo_box.dart

```diff
@@ -1072,16 +1072,16 @@ class _ComboboxState<T> extends State<Combobox<T>> with WidgetsBindingObserver {
       ),
     };
     focusNode!.addListener(_handleFocusChanged);
-    final FocusManager focusManager = WidgetsBinding.instance!.focusManager;
+    final FocusManager focusManager = WidgetsBinding.instance.focusManager;
```

Comment:
If I would replace the `!` by `?` then the type of the variable would be `FocusManager?` instead of `FocusManager`, right?. I don't think that's desirable as I would require lots of more changes...

Reply:
I believe it'd be something like

```suggestion
    // ignore: invalid_null_aware_operator
    final FocusManager focusManager = WidgetsBinding.instance?.focusManager!;
```

Altough, we make use of `TextSelectionControls`, in which its API has changed. Then, I believe it's not possible to keep the backwars compatibility :/